### PR TITLE
[Series] Implementation

### PR DESCRIPTION
**Pull request by habere-et-dispertire**
[Agreement.](http://forum.exercism.org/t/new-exercise-contributions/4077/3)

### CODE REVIEW

File: exercises/practice/solutions/series.rakutest [hunk 1]
```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env raku
+use Test;
+use lib $?FILE.IO.dirname;
+use Series;
+
+cmp-ok( # begin: 7ae7a46a-d992-4c2a-9c15-a112d125ebad
+    series("1", 1),
+    "eqv",
+    $["1"],
```
[the remark this answers is not in the document]
Comment (habere-et-dispertire) [reply exercises/practice/solutions/series.rakutest:9]: Should all the `.raku` calls be prefixed with decontainerization ?

PS. The files are all over the place — so sorry — I'll fix that. That's the first and last time I try drag and drop on GitHub ! I was just trying to speed things up ! _Sigh._

[the remark this answers is not in the document]
Comment (m-dango) [reply exercises/practice/solutions/series.rakutest:9]: It shouldn't be necessary for all of them. Usually the ones that could use decontainerization are contents of arrays.

**Comment by m-dango** [comment]
The files are currently in the main `exercises/practice` dir, rather than their own `series` dir.

File: exercises/practice/solutions/series.rakutest (hunk 1 above)
[the remark this answers is not in the document]
Comment (habere-et-dispertire) [reply exercises/practice/solutions/series.rakutest]: I put one in on the expected result `%case<expected>.<>.raku`

**Comment by habere-et-dispertire** [comment]
I moved all the files misplaced by GitHub Drag & Drop.  Thank you for your patience.

File: exercises/practice/series/series.rakutest [hunk 2]
```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env raku
+use Test;
+use lib $?FILE.IO.dirname;
+use Series;
+
+cmp-ok( # begin: 7ae7a46a-d992-4c2a-9c15-a112d125ebad
+    series("1", 1),
+    "eqv",
+    ["1"],
```
Comment (m-dango) [inline exercises/practice/series/series.rakutest:9]: I'd personally prefer a list for the sake of consistency. I tend to choose lists over arrays if I have no intention of assigning/modifying the contents: https://github.com/exercism/raku/blob/main/exercises/practice/all-your-base/all-your-base.rakutest

**Comment by habere-et-dispertire** [comment]
I changed to a smartmatch to allow for greater equivalence as that feels more rakuish. What do you feel about putting [`!!!`](https://docs.raku.org/routine/!!!) in the solution stub ?

File: exercises/practice/series/series.rakutest (hunk 2 above)
Comment (habere-et-dispertire) [reply exercises/practice/series/series.rakutest]: Agreed and completed.

**Review by m-dango — APPROVED** [review]

**Comment by habere-et-dispertire** [comment]
> Putting `!!!` in stubs is something I've considered in the past, but I don't think its user friendly on the Exercism platform.

Shall we dispense of it in the [robot exercise](https://github.com/exercism/raku/blob/main/exercises/practice/robot-name/.meta/template-data.yaml) then ?

~~~yaml
stub: |-
  has Str:D $.name = '';

  method reset-name {!!!}
~~~

**Pull request merged**